ubiquibot-config-default.yml

```diff
@@ -0,0 +1,41 @@
+---
+chain-id: 1
+base-multiplier: 1000
+issue-creator-multiplier: 2000
+time-labels:
+  - name: "Time: <1 Hour"
+    weight: 0.125
+    value: 3600
+  - name: "Time: <1 Day"
+    weight: 1
+    value: 86400
+  - name: "Time: <1 Week"
+    weight: 2
+    value: 604800
+  - name: "Time: <2 Weeks"
+    weight: 3
+    value: 1209600
+  - name: "Time: <1 Month"
+    weight: 4
+    value: 2592000
+priority-labels:
+  - name: "Priority: 0 (Normal)"
+    weight: 1
+  - name: "Priority: 1 (Medium)"
+    weight: 2
+  - name: "Priority: 2 (High)"
+    weight: 3
+  - name: "Priority: 3 (Urgent)"
+    weight: 4
+  - name: "Priority: 4 (Emergency)"
+    weight: 5
+auto-pay-mode: true
+analytics-mode: true
+incentive-mode: false
```

Comment:
Should we rename this to `comment-incentives` ?

Is that what its referring to?

Reply:
```suggestion
comment-incentives: true
```

Reply:
@0xcodercrane

Reply:
All bot params with the `-mode` suffix are [meant](https://github.com/ubiquity/ubiquibot/blob/development/src/bindings/config.ts#L59) to be booleans. So this was kind of an original design to put boolean params in a [single place](https://github.com/ubiquity/ubiquibot/blob/development/src/bindings/config.ts#L59) and suffix them with `-mode`

Reply:
I just don't like the idea of having redundant parts in property names. 

I generally like to avoid redundancy.

Reply:
@rndquu can we remove `-mode` or are there any other reasons to share? I'm about to make a new bounty for all of these changes.